docker-compose.prod.yml

```diff
@@ -4,28 +4,22 @@
 version: '3.6'
 
 services:
-  # Build and run the middleware service from the Dockerfile, without mounting
-  # a volume for development
   middleware:
     image: fordtotalloss/middleware
-    build: ./middleware
-    environment:
-      NODE_ENV: production
+    build:
+      context: .
+      dockerfile: ./middleware/Dockerfile
 
   web:
     image: fordtotalloss/web
-    build: ./frontend
+    build:
+      context: .
+      dockerfile: ./frontend/Dockerfile
     ports:
-      - 80:80
-      - 443:443
-    environment:
-      NODE_ENV: production
+      - 4200:80
```

Comment:
So upon starting the app we can use ports 4200-4280 as different nodes?

Reply:
No. This maps port 4200 to port 80 of the container.

So you can run `localhost:4200` like you normally do to access it.

Reply:
App is being served by nginx on port 80 in production. In development it's being served by angular's dev server on port 4200.